test/Microsoft.ML.Tests/Transformers/KeyToBinaryVectorEstimatorTest.cs

```diff
@@ -65,6 +65,13 @@ public void KeyToBinaryVectorStatic()
                 VectorString: ctx.LoadText(1, 4)
             ));
 
+            using (var stream = default(Stream))
+            {
+                var mlContext = new MLContext();
+                var trainedModel = mlContext.Model.Load(stream, out var inputSchema);
+                var = mlContext.Model.CreatePredictionEngine<int, int>(trainedModel, inputSchemaDefinition: inputSchema);
```

Comment:
compile error here

Reply:
Actually, I assume you don't want to make this whole change, right?

---
In reply to: [274477319](https://github.com/dotnet/machinelearning/pull/3288#discussion_r274477319) [](ancestors = 274477319)

Reply:
> Actually, I assume you don't want to make this whole change, right?
> 
> In reply to: [274477319](https://github.com/dotnet/machinelearning/pull/3288#discussion_r274477319) [](ancestors = 274477319)

Um, yeah. Sorry. :) I'll nuke this from the commit history, but address the other comments separately.